src/main/java/com/ksm/wstbackoffice/service/CountryService.java

```diff
@@ -21,6 +22,7 @@ public CountryService(CountryRepository countryRepository, CountryMapper country
     }
 
     public CountryDto findById(String id) {
-        return countryMapper.toDto(countryRepository.findById(id).orElse(null));
+        return countryMapper.toDto(countryRepository.findById(id)
```

Comment:
This is a good time to learn or improve unit testing.
Please write unit tests.
It should check both positive results and exception throwing.